Documentation/meson.build

```diff
@@ -4,6 +4,8 @@ if not hotdoc_p.found()
     subdir_done()
 endif
 
+run_command('git', 'clone', 'https://github.com/nnstreamer/nnstreamer-example', 'nnst-exam')

```

Comment:
Need to check whether it exists or not. BTW, is this used only internally? `pdebuild` or `gbs` cannot run this command.

Reply:
I changed the hotdoc document generation from meson build to bash script.
Users who want to generate documents can use the scripts.